ISISComputingGroup/ibex_gui
# Ticket1016 tidy up writables

Changes to the classes are shown here https://github.com/ISISComputingGroup/ibex_developers_manual/blob/master/refactoring_for_observables_and_writers.rst.

To test:
*Check unit tests pass
*Do some random testing around observables in the UI, especially changing the synoptic, to ensure nothing is broken
*Check updates are shown in this document https://github.com/ISISComputingGroup/ibex_developers_manual/blob/master/refactoring_for_observables_and_writers.rst
